runtime/doc/vim_diff.txt

```diff
@@ -218,6 +220,7 @@ Input/Mappings:
 
 Normal commands:
   |gO| shows a filetype-defined "outline" of the current buffer.
+  |Q|  replays the last recorded macro.
```

Comment:
I don't think this needs to appear in `vim_diff.txt` twice. That one in changes features section is enough.

Reply:
removed in 1eb689ac72c2a3c9744286484b255087151be7cf